### PR TITLE
Improve logger messages

### PR DESCRIPTION
**Pull request by rojserbest**

### CODE REVIEW

**Comment by buddhhu** [comment]
Something useful will be a better pr ig🌝

**Comment by TheHamkerCat** [comment]
lol

**Comment by rojserbest** [comment]
@TheHamkerCat you were not supposed to say that. See the diff.

**Comment by MarshalX** [comment]
Why don't you want to use infinitives?

**Comment by rojserbest** [comment]
@MarshalX because you're not talking about something that is 100% done, but you talk about something is about to be tried to be done?

**Comment by MarshalX** [comment]
> @MarshalX because you're not talking about something that is 100% done, but you talk about something is about to be tried to be done?

To mark long action I use double and triple dots in the end of log message…

**Comment by rojserbest** [comment]
> > @MarshalX because you're not talking about something that is 100% done, but you talk about something is about to be tried to be done?
> 
> To mark long action I use double and triple dots in the end of log message…

So how is it related to using infinitive forms? i think the current way is cooler with triple dots. But it's what you say.

**Comment by MarshalX** [comment]
@rojserbest Continuous term use obliges to always have an expiration message (thst something was end). The infinitive is not obliges

**Comment by rojserbest** [comment]
@MarshalX

File: pytgcalls/pytgcalls/implementation/group_call_native.py [hunk 1]
```diff
@@ -379,27 +379,27 @@ def restart_recording(self):
         self.__native_instance.restartAudioOutputDevice()
 
     def __participant_descriptions_required_callback(self, ssrcs_list: List[int]):
-        logger.debug('Participant descriptions required..')
+        logger.debug('Participant descriptions required.')
```
Comment (MarshalX) [inline pytgcalls/pytgcalls/implementation/group_call_native.py:382]: So there is the dot in the end of row

File: pytgcalls/pytgcalls/implementation/group_call_native.py [hunk 2]
```diff
@@ -379,27 +379,27 @@ def restart_recording(self):
...
 
         using_cache = True
         requested_participants = list()
 
         for ssrc in ssrcs_list:
             requested_participant = self.__cached_participants.get(ssrc)
             if requested_participant is None:
-                logger.debug('Can\'t find participant in cached list')
+                logger.debug('Couldn\'t find participant in cached list')
```
Comment (MarshalX) [inline pytgcalls/pytgcalls/implementation/group_call_native.py:390]: But why there isn’t a dot?

Comment (rojserbest) [reply pytgcalls/pytgcalls/implementation/group_call_native.py]: Done!

**Comment by MarshalX** [comment]
Pls review group call native file. There is log messages without dots in the end. Any double dots instead of triple

**Comment by MarshalX** [comment]
logger.debug(f'Not equal ssrc. Expected: {ssrc}. Actual: {self.mtproto_bridge.my_ssrc}')

**Comment by MarshalX** [comment]
logger.debug('Duplicate SSRC')

**Comment by MarshalX** [comment]
Rly?

**Comment by rojserbest** [comment]
> Rly?

Yes, rly. :)

**Pull request merged**